docs/source/index.rst

```diff
@@ -31,12 +31,12 @@ architectures, and common image transformations for computer vision.
    :maxdepth: 2
    :caption: Package Reference
 
+   models
+   transforms
    datasets
+   ops
    io
-   models
    feature_extraction
```

Comment:
I think We should club this with models, just after it.

Reply:
> We should club this with models

I agree, I think eventually we will move it into the "Models and pre-trained weights" section. It is `torchvision.models.feature_extraction` after all. But since the models section needs a much bigger revamp, I wanted to leave this out for now